content/p/vulnerability-disclosure-policy.md

```diff
@@ -0,0 +1,72 @@
+- Please also include your recommended remediation(s), if any, or any other concerns.
+
+> #### Do Not Send:
+> - Sensitive or personal information.
+> - Public disclosures of vulnerabilities.
+
+Our maintainers will attempt to respond to and confirm your report within 2-3 days, but if you believe your report to be *critical* to user safety and security, please note as such in the subject. We are fortunate enough to have hundreds of thousands of systems relying on the expertise of the AlmaLinux OS Team, and we take security very seriously.
+
+## Example Report
+
+```text
+- **Title**: Failed to start OSBuild Composer
+
+- **Environment**: AlmaLinux 8, Platform: X86_64, OS Version: 8.5
+
+- **Description**:
+    I am unable to start the OS build composer. When I try to start the `osbuild-composer.service`, it stops immediately:
+
+    [root@localhost ~]# systemctl status osbuild-composer.service
+    ● osbuild-composer.service - OSBuild Composer
+        Loaded: loaded (/usr/lib/systemd/system/osbuild-composer.service; enabled; vendor preset: disabled)
+        Active: failed (Result: exit-code) since Tue 2022-02-08 00:18:43 EST; 32min ago
+        Process: 1657 ExecStart=/usr/libexec/osbuild-composer/osbuild-composer (code=exited, status=1/FAILURE)
+        Main PID: 1657 (code=exited, status=1/FAILURE)
+
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: osbuild-composer.service: Failed with result 'exit-code'.
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: osbuild-composer.service: Service RestartSec=100ms expired, scheduling restart.
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: osbuild-composer.service: Start request repeated too quickly.
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: Failed to start OSBuild Composer.
+
+- **Steps to Reproduce**:
+    1. Install AlmaLinux 8.4 and update to 8.5.
+    2. Run `sudo dnf install osbuild-composer composer-cli cockpit-composer bash-completion`.
+    3. Run `sudo systemctl enable --now osbuild-composer.socket`.
+    4. Try running `composer-cli blueprint list` or start the image builder service in Cockpit.
+
+- **Expected Result**: The OS build composer should start without issues.
+
+- **Actual Result**: `osbuild-composer.service` stops immediately with an exit code error.
+
+- **Severity**: Urgent
+```
+
+## Where to Report
+
+- For any issue that requires a coordinated release, send your report to [security@almalinux.org](mailto:security@almalinux.org) directly so we can coordinate a responsible patch and release.
+
+- For issues that are directly related to the AlmaLinux operating system itself and do not require coordinated disclosure, please send your report to [bugs.almalinux.org](https://bugs.almalinux.org), this ensures that your report is received by the right people.
+
+- For **non OS-related** reports (.e.g Elevate, almalinux.org website, etc), open an issue on the GitHub repo for that part of the project. 
```

Comment:
Which github repo(s)? can we provide a link here too?

Reply:
I think people that want to report about part of the project would have to find the respective GitHub repo.

Reply:
Sure, maybe I wasn't clear enough, my bad. Since they need to find the right repo, we can provide a link to our github organization, or to https://github.com/orgs/AlmaLinux/repositories. What do you think?